spotbugs/src/main/java/edu/umd/cs/findbugs/util/Archive.java

```diff
@@ -70,9 +70,4 @@ private static String getExtension(String fileName) {
         String extension = fileName.substring(lastDot).toLowerCase(Locale.ENGLISH);
         return extension;
     }
-
-    public static boolean isLibraryFileName(String fileName) {
```

Comment:
I think it's not so good to remove a public API by minor/patch update, could you revert this change and add `@Deprecated` annotation instead?

If you make a GitHub issue to remove this deprecated in the next major update, I will add it into the [milestone for v5](https://github.com/spotbugs/spotbugs/milestone/15).